src/environment/dev/app/Dockerfile

```diff
@@ -0,0 +1,67 @@
+FROM composer/composer:2.5.7-bin as composer-bin
+
+FROM node:20.2.0-bullseye-slim as node
+
+FROM php:8.2.5-fpm-bullseye
+
+COPY --from=composer-bin ./composer /usr/bin/composer
+
+ARG USER_NAME=host-user
+ARG USER_ID=1000
+ARG PHP_FPM_GROUP=www-data
+
+RUN adduser \
+        --disabled-password \
+        --uid ${USER_ID} \
+        ${USER_NAME} \
+    && usermod \
+        --append \
+        --groups \
+        ${PHP_FPM_GROUP} \
+        ${USER_NAME}
+
+#Add node and npm
+COPY --from=node --chown=${USER_NAME}:root /usr/local/lib/node_modules /usr/local/lib/node_modules
+COPY --from=node --chown=${USER_NAME}:root /usr/local/bin/node /usr/local/bin/node
+RUN ln --symbolic /usr/local/lib/node_modules/npm/bin/npm-cli.js /usr/local/bin/npm \
+    && chown --no-dereference ${USER_NAME}:root /usr/local/bin/npm \
+    && ln --symbolic /usr/local/lib/node_modules/npm/bin/npx-cli.js /usr/local/bin/npx \
+    && chown --no-dereference ${USER_NAME}:root /usr/local/bin/npx
+
+# Use the default development configuration
+RUN mv "${PHP_INI_DIR}/php.ini-development" "${PHP_INI_DIR}/php.ini"
+
+# For other versions check: http://nginx.org/packages/mainline/debian/pool/nginx/n/nginx/
+ARG NGINX_VERSION="1.25.0-1~bullseye"
+ARG PHPREDIS_VERSION=5.3.7
```

Comment:
`6.0.2` is the latest at this moment.

https://pecl.php.net/package/redis

Reply:
updated